File: Fast_bookapi/app/src/main/java/com/example/fast_bookapi/room/History.java
package com.example.fast_bookapi.room;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class History {

    public History(Integer uid, String keyword) {
        this.uid = uid;
        this.keyword = keyword;
    }

    @PrimaryKey
    private Integer uid;

    @ColumnInfo(name = "keyword")
    private String keyword;

    public Integer getUid() {
        return uid;
    }

    public void setUid(Integer uid) {
        this.uid = uid;
    }

    public String getKeyword() {
        return keyword;
    }

    public void setKeyword(String keyword) {
        this.keyword = keyword;
    }
}

File: Fast_calculator/app/src/main/java/com/example/fast_calculator/model/History.kt
package com.example.fast_calculator.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

//Room : History 클래스 자체를 데이터베이스의 테이블로 사용하겠다
@Entity
data class History(
    @PrimaryKey val uid: Int?,
    @ColumnInfo(name = "expression") val expression: String?,
    @ColumnInfo(name = "result") val result: String
)